template.md

```diff
@@ -10,6 +10,21 @@
 - **Location**: CITY, COUNTRY
 - **Attendee information**: LINK TO REFLECTOR
 
+```mermaid
+---
+displayMode: compact
+---
+gantt
+    title: TBD Meeting
+    dateFormat  YYYY MM DD HH mm Z
+    tickInterval 3d
```

Comment:
```suggestion
```

`tickInterval` syntax is incorrect (which is why it is displaying default of ticks at `1day`.  I don't think you would want 3day ticks, so just remove.